schemas/2016-05-01-preview/Microsoft.MachineLearning.json

```diff
@@ -383,9 +379,7 @@
           "oneOf": [
             {
               "type": "object",
-              "additionalProperties": {
-                "type": "object"
-              }
```

Comment:
Yes, "globalParameters" will still be a dictionary. With the previous schema, it was a dictionary of string -> JSON objects. With this new schema, its now a dictionary of string -> any JSON type.